### ISSUE TITLE
[NTR] add "method"

### ISSUE THREAD

**Issue by Zack-83** [NFDI4Chem/VibrationalSpectroscopyOntology#28]
For new term requests, please provide the following information:

## Preferred term label

Raman spatial method

## Synonyms

method

## Textual definition

The "spatial coverage" of the Raman spectroscopy / microscopy experiment. Admitted values: "Single spectrum"; "Line scan/mapping"; "Vertical mapping"; "2D mapping"; "3D mapping".

## Suggested parent term

Please look in the hierarchy in a browser such as the [NFDI4Chem TS](https://terminology.nfdi4chem.de/ts/ontologies/vibso)

## Examples

Please provide a more detailed description that examplifies how the term is supposed to be used, in terms of what it is suppose to repesent (e.g. a concept or a relation).

## Attribution

ORCID:0000-0001-7694-5519

**Comment by StroemPhi** [comment]
@Zack-83 can you please provide more context as to what you mean with this exactly, maybe links to papers? Also have you checked in the [Raman spectroscopy branch](https://terminology.nfdi4chem.de/ts/ontologies/vibso/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHMO_0000656) we import from CHMO in VIBSO, if these "methods" are defined there as special kinds of Raman spectroscopy? For me the question would be if you are refering to a specialization/kind of Raman spectroscopy or to a special kind of output generated by a certain Raman spectroscopy, so focusing on the generated spectrum.

**Comment by StroemPhi** [comment]
If you refer to the former, a special kind of Raman spectroscopy, we would have to create a new subclass of either [chmo:Raman spectroscopy](https://terminology.nfdi4chem.de/ts/ontologies/vibso/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHMO_0000656 )or [chmo:laser Raman spectroscopy](https://terminology.nfdi4chem.de/ts/ontologies/vibso/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHMO_0002515), which is probably similar to [chmo:two-dimensional infrared spectroscopy](https://terminology.nfdi4chem.de/ts/ontologies/vibso/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHMO_0002710).

**Comment by Zack-83** [comment]
I checked. It seems the spatial dimension is not considered in CHMO.  
I will ask @RainerStosch or @Stefano-Luin if other aspects are relevant to distinguish among techniques

**Comment by Zack-83** [comment]
We have renamed the class and the admitted values to make it clear, that we are only taking about the spatial component.

This spatial information can be registered independently from the subclasses defined in https://terminology.nfdi4chem.de/ts/ontologies/vibso/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHMO_0000656

Concerning the subclasses, we have two questions: who defined them and with which criteria? That classification mixes up different aspects: source, excitation band, enhancement, sample preparation... At least the sample preparation should not be considered, because it has nothing to do with the analytical method used after that.

We can help sorting the chaos :)

**Comment by StroemPhi** [comment]
Regarding the existing children of [CHMO:Raman spectroscopy](https://terminology.nfdi4chem.de/ts/ontologies/chmo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHMO_0000656), all I can say is that we would have to contact @batchelorc or better file an issue in CHMO directly to find out more about their history. My hope was that we can reuse as many of these subclasses as possible and mint new ones only where this is not possible. And once VIBSO is in a more mature state we would have to speak to Colin and the OBI devs anyway to finally decide where should live what. But I think it is best to have a working proposal with VIBSO that can be constructivly discussed with all necessary ontology stakeholders.

**Comment by StroemPhi** [comment]
![grafik](https://user-images.githubusercontent.com/56996739/217841586-9e4c8a1d-4981-4566-8da3-303417440809.png)
[Raman Spectroscopy for Chemical Analysis, p. 294ff](https://onlinelibrary.wiley.com/doi/book/10.1002/0471721646)
Are you talking about this?

**Comment by Zack-83** [comment]
Yes, it goes in the same direction. But we foresee a third dimension z, which produces three more cases (vertical scan, vertical surface, volume)

**Comment by Zack-83** [comment]
Hallo Philip,

we can live with this classification, with minor changes: 

>>METHODS<<

* **IAO:plan specification**
    * **Raman imaging mode**
        -- synonym: Raman acquisition mode
        -- def.: A plan specification that specifies the spatial coverage of a Raman spectroscopy experiment, by determining whether to produce a Raman image of the region of interest (ROI) from a single acquisitions or instead from multiple mapped scans.
        * **Raman single acquisition mode**
            -- def: A Raman acquisition mode that specifies to cover the region of interest (ROI) at once, by focusing the laser beam to cover the whole ROI. 
            -- comment: The spatial resolution of a Raman image produced with this mode is determined by the magnification of the optics in the system and the size of the elements in the detector.
        * **Raman mapping mode**
        -- def.: A Raman acquisition mode that specifies to scan the region of interest (ROI) sequentially, by displacing a focused steady laser beam narrower than the ROI at different locations on the sample within an equally spaced grid spanning the ROI.  
        -- comment: The spatial resolution of a Raman image produced with this mode is determined by a combination of the laser focus area and the spacing between the acquisition points in the scanning grid.
            * **point-by-point mapping mode**
                -- def.: A mapping mode in which the laser beam is specifed to be focused to a spot.
            * **line focus mapping mode**
                -- def.: A mapping mode in which the laser beam is specifed to be focused to a line.

>>RESULTS<<

* OBI:planned process
    * OBI:assay
        * **Raman map**
            -- def.: An assay implementing a mapping mode, in which the signal intensity or signal energy (band position) of a particular Raman vibrational mode or the intensity ratio of two Raman vibrational modes are mapped by undertaking Raman spectroscopy (2.5.17) while pointwise scanning the sample surface and subsequently evaluating each pixel of the image.
            -- source: ISO 18115-3 2.5.14
              * Raman horizontal 1D map
              * Raman vertical 1D map
              * Raman horizontal 2D map
              * Raman vertical 2D map
              * Raman 3D map

**Comment by StroemPhi** [comment]
The assay type Raman mapping was already defined in CHMO, where it is called 'Raman microscopy' and 'Raman mapping' as well as 'Raman imaging' are provided as synonyms. We thus imported this class from CHMO (see: #127)
